sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/parser/AstBuilder.scala

```diff
@@ -1426,6 +1428,12 @@ class AstBuilder(conf: SQLConf) extends SqlBaseBaseVisitor[AnyRef] with Logging
         Minute(expression(ctx.source))
       case "SECOND" =>
         Second(expression(ctx.source))
+      case "MILLISECONDS" | "MSEC" =>
+        Milliseconds(expression(ctx.source))
+      case "MICROSECONDS" | "USEC" =>
```

Comment:
ditto. Maybe, `US` and `USECS` are missing here.

Reply:
Why? Are they used somewhere in ported PostgreSQL tests?